plugins/metrics/mana.go

```diff
@@ -64,6 +74,65 @@ func ConsensusManaMap() mana.NodeMap {
 	return result
 }
 
+// AverageNeighborsAccess returns the average access mana of the nodes neighbors.
+func AverageNeighborsAccess() float64 {
+	return averageNeighborsAccess.Load()
+}
+
+// AverageNeighborsConsensus returns the average consensus mana of the nodes neighbors.
+func AverageNeighborsConsensus() float64 {
+	return averageNeighborsConsensus.Load()
+}
+
+// AveragePledgeAccessMap returns the average pledged access mana of the node.
+func AveragePledgeAccessMap() mana.NodeMap {
+	accessPledgeLock.RLock()
+	defer accessPledgeLock.RUnlock()
+	result := mana.NodeMap{}
+	for k, pledges := range accessPledge {
+		sum := 0.0
+		for _, v := range pledges {
+			sum += v
+		}
+		result[k] = sum / float64(len(pledges))
+	}
+	return result
+}
+
+// AveragePledgeConsensusMap returns the average pledged consensus mana of the node.
+func AveragePledgeConsensusMap() mana.NodeMap {
+	consensusPledgeLock.RLock()
+	defer consensusPledgeLock.RUnlock()
+	result := mana.NodeMap{}
+	for k, pledges := range consensusPledge {
+		sum := 0.0
+		for _, v := range pledges {
+			sum += v
+		}
+		result[k] = sum / float64(len(pledges))
+	}
+	return result
+}
+
+// addPledge populates the pledge logs.
+func addPledge(event *mana.PledgedEvent) {
+	amount := (event.AmountBM1 + event.AmountBM2) / 2
```

Comment:
This `/2` should actually be a function of the default weight we use for mana calculation. It is 0.5 atm, but what if we change it after testing?
Alternatively, we could say that we need this metric to visualize average base mana pledge amounts, then we decouple from mana calculation and weight. We end up with a graph that shows how much base mana was pledged to a node on average (BM1 and BM2 separately in the same graph). Because of mana calculation, it doesn't sound right to equate pledge amounts to calculated mana values.